### PR TITLE
Update environment

### PR DESCRIPTION
**Pull request by PerilousApricot**
Use CVMFS for software, where possible

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -8,29 +8,25 @@ if [[ $HOSTNAME == "login.snowmass21.io" ]]; then
...
-    fi
+# else
+#     which root >> /dev/null
+#     if [[ $? -ne 0 ]]; then
+# 	echo "Please ensure ROOT is available and retry"
+# 	exit
+#     fi
 fi
 
+source /cvmfs/sft.cern.ch/lcg/releases/LCG_99/ROOT/v6.22.06/x86_64-centos7-gcc8-opt/ROOT-env.sh
```
Comment (PerilousApricot) [inline setup.sh:27]: do we need the ROOT import here? it should come with the LCG_101 view

File: makeGridPacks.py [hunk 2]
```diff
@@ -112,7 +112,7 @@
 f=open(sample+'.mg','w')
 if __name__=='__main__':
     f.write(definitions+'\n')
-    f.write('set lhapdf_py3 %s/bin/lhapdf-config\n'%os.environ['prodBase'])
+    f.write('set lhapdf_py3 /cvmfs/sft.cern.ch/lcg/views/LCG_99/x86_64-centos7-gcc8-opt/bin/lhapdf-config\n')
```
Comment (PerilousApricot) [inline makeGridPacks.py:115]: should be LCG_101

File: setup.sh (hunk 1 above)
Comment (jstupak) [reply setup.sh:27]: im not sure.  I'll remove it and see if it causes trouble

Comment (jstupak) [reply setup.sh:27]: and note, I was using LCG_99 in my previous testing, so I stuck with that.  Is there any reason to use 101 instead?

File: makeGridPacks.py (hunk 2 above)
Comment (jstupak) [reply makeGridPacks.py]: done

File: setup.sh (hunk 1 above)
Comment (jstupak) [reply setup.sh]: Ignore my last comment, I switched to 101

**Pull request merged**